### PR TITLE
Add missing CHANGELOG entries

### PR DESCRIPTION
**Pull request by daboross**
Fixes one typo from #200, and adds changelog entries for all PRs since the last release.

### CODE REVIEW

**Comment by daboross** [comment]
Rebased on #201

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
-- Change `LocalRoomName`'s orientation so that north to be negative and south is positive.
+- Change `LocalRoomName`'s orientation so that north is negative and south is positive.
   This is a breaking change for `LocalRoomName::from_coords`, fields and add/sub implementations
+  (#200)
+- Work around bug where IntelliJ-Rust didn't understand `screeps::game::*` modules created by a
+  macro, even with experimental engine enabled (#197)
+- `OwnedStructureProperties`'s `my` method now correctly handles the value being undefined.
+  This fixes a panic on checking `my` for unowned controllers (#189)
+- `StructurePortal` no longer implements `OwnedStructure` and `Attackable`. (#190)
+- Collections provided by `Game` now implements the `hashmap` method to retrieve both keys
```
Comment (ASalvail) [inline CHANGELOG.md]: I made a typo here:
implement~s~
also, `hashmap` isn't a method.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+  and values at once. (#194)
+- Fix prices returned from `game::market` APIs being integers rather than floats (#179)
```
Comment (ASalvail) [inline CHANGELOG.md]: This is also breaking (if we're keeping track)

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,14 +4,14 @@ Unreleased
...
   This is a breaking change for `LocalRoomName::from_coords`, fields and add/sub implementations
   (#200)
+- Fix prices returned from `game::market` APIs being integers rather than floats (breaking) (#179)
 - Work around bug where IntelliJ-Rust didn't understand `screeps::game::*` modules created by a
   macro, even with experimental engine enabled (#197)
 - `OwnedStructureProperties`'s `my` method now correctly handles the value being undefined.
   This fixes a panic on checking `my` for unowned controllers (#189)
 - `StructurePortal` no longer implements `OwnedStructure` and `Attackable`. (#190)
-- Collections provided by `Game` now implements the `hashmap` method to retrieve both keys
+- Collections provided by `Game` now implements the `hashmap` tunfction to retrieve both keys
```
Comment (ASalvail) [inline CHANGELOG.md]: Gotta love those tunfction 😛

Comment (ASalvail) [reply CHANGELOG.md]: and implement still has one too many s

Comment (daboross) [reply CHANGELOG.md]: :pencil2: fixed!

**Pull request merged**